content/post/benchmarking-gnns/index.md

```diff
@@ -0,0 +1,249 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Benchmarking Graph Neural Networks"
+subtitle: ""
+summary: ""
+authors: [vijay-dwivedi, chaitanya-joshi, xavier-bresson]
```

Comment:
As the initiative to create this blog was Vijay's, I feel we should give credit where credit's due! (Unless the rest of us are able to make real contributions to it :) )